README.md

```
@@ -9,19 +9,43 @@
+.\install.ps1
 ```
 
-**Step 2 — Install a domain plugin**:
+Restart Claude Code after running — all 13 skills and agents will be active.
+
+### Option B — Marketplace (install plugins individually)
+
+If you prefer the built-in marketplace installer, add the marketplace once then
+install each plugin **one at a time** to avoid the race condition:
+
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)